modules/googleServices/routes/googleServices.ts

```diff
@@ -0,0 +1,32 @@
+import * as express from 'express';
+import { getGeoreferencia, autocompletar } from '../../../utils/serviciosGoogle';
+
+const router = express.Router();
+
+router.post('/getGooglePoint', async (req, res, next) => {
```

Comment:
La ruta debería llamarse `georeferenciar`, por lo meno no incluir la palabra `get`.
El metodo de la request debería ser `GET`.
```suggestion
router.get('/georeferenciar', async (req, res, next) => {
```